plugins/revealer/qt.py

```diff
@@ -0,0 +1,688 @@
+
+
+    def customtxt_limits(self):
+        txt = self.text.text()
+        self.max_chars.setVisible(False)
+        self.char_count.setText("("+str(len(txt))+"/216)")
+        if len(txt)>0:
+            self.ctext.setEnabled(True)
+        if len(txt) > 216:
+            self.text.setPlainText(self.text.toPlainText()[:216])
+            self.max_chars.setVisible(True)
+
+    def t(self):
+        self.txt = self.text.text()
+        self.seed_img(is_seed=False)
+
+    def cypherseed_dialog(self, window):
+
+        d = WindowModalDialog(window, "Revealer")
+        d.setMinimumWidth(420)
+
+        self.c_dialog = d
+
+        self.vbox = QVBoxLayout(d)
+        self.vbox.addSpacing(21)
+
+        logo = QLabel()
+        self.vbox.addWidget(logo)
+        logo.setPixmap(QPixmap(':icons/revealer.png'))
+        logo.setAlignment(Qt.AlignCenter)
+        self.vbox.addSpacing(42)
+
+        grid = QGridLayout()
+        self.vbox.addLayout(grid)
+
+        cprint = QPushButton(_("Generate encrypted seed PDF"))
+        cprint.clicked.connect(partial(self.seed_img, True))
+        self.vbox.addWidget(cprint)
+        self.vbox.addSpacing(14)
+
+        self.vbox.addWidget(WWLabel(_("and/or type any secret below:")))
+        self.text = ScanQRTextEdit()
+        self.text.setTabChangesFocus(True)
+        self.text.setMaximumHeight(70)
+        self.text.textChanged.connect(self.customtxt_limits)
+        self.vbox.addWidget(self.text)
+
+        self.char_count = WWLabel("")
+        self.char_count.setAlignment(Qt.AlignRight)
+        self.vbox.addWidget(self.char_count)
+
+        self.max_chars = WWLabel("<font color='red'>" + _("This version supports a maximum of 216 characters.")+"</font>")
+        self.vbox.addWidget(self.max_chars)
+        self.max_chars.setVisible(False)
+
+        self.ctext = QPushButton(_("Generate custom secret encrypted PDF"))
+        self.ctext.clicked.connect(self.t)
+
+        self.vbox.addWidget(self.ctext)
+        self.ctext.setEnabled(False)
+
+        self.vbox.addSpacing(21)
+        self.vbox.addLayout(Buttons(CloseButton(d)))
+        return bool(d.exec_())
+
+
+    def update_wallet_name (self, name):
+        self.wallet_name = str(name)
+        self.base_name = self.base_dir + self.wallet_name
+        
+    def seed_img(self, is_seed = True):
+      
+        if not self.cseed and self.txt == False:
+            return
+        
+        if is_seed:
+            txt = self.cseed
+        else:
+            txt = self.txt.upper()
+
+        img = QImage(self.size[0],self.size[1], QImage.Format_Mono)
+        bitmap = QBitmap.fromImage(img, Qt.MonoOnly)
+        bitmap.fill(Qt.white)
+        painter = QPainter()
+        painter.begin(bitmap)
+        QFontDatabase.addApplicationFont(os.path.abspath('plugins/revealer/SourceSansPro-Bold.otf'))
```

Comment:
This won't work with binaries.
Maybe `os.path.dirname(__file__)+'/SourceSansPro-Bold.otf'`? @bauerj